mods/ra/rules/defaults.yaml

```diff
@@ -466,7 +466,7 @@
 	ExternalCapturable:
 	ExternalCapturableBar:
 	EmitInfantryOnSell:
-		ActorTypes: e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,c1,c2,c3,c4,c5,c6,c7,c8,c9,c10,e6,e6,e6,e6,e6
+		ActorTypes: e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,e1,tecn,tecn,tecn,tecn,tecn,tecn,tecn,tecn,tecn,tecn,e6,e6,e6,e6,e6
```

Comment:
This list now contains 15x e1, 10x tecn and 5x e6, so you can actually shorten it drastically to `e1,e1,e1,tecn,tecn,e6`.

Although on that note, only construction yards should emit `e6`, so maybe we should remove it here and only add it on FACT.